slackcli/names.py

```diff
@@ -28,6 +28,14 @@ def username(self, user_id):
             self.user_index[user_id] = slack.client().users.info(user_id).body['user']['name']
         return self.user_index[user_id]
 
+    def user_id(self, username):
+        members = slack.client().users.list().body['members']
+        member = [x for x in members if x['name'] == username][0]
```

Comment:
This will fail with a KeyError if the user is not found in the results. Instead, you want to write something like:

```
for member in members:
    if member["name"] == username:
        return member["id"]
return None
```